task_processing/task_processing/docker_task_configs.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+class DockerTaskConfigs(object):
```

Comment:
dict is even better though might be less convenient (['blah'] instead of .blah), also namedtuple will freak out on typos while dict won't. i'm fine with both.